domains/thikhathali.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Thikhathali",
+        "email": "thikhamataboge@gmail.com"
+    },
+    "record": {
+        "CNAME": "thikhathali.github.io"
```

Comment:
Cant access the website (says that page doesnt exist)

Reply:
Rename the github repo that your website is in to thikhathali.github.io

Reply:
Just did.